src/context.rs

```diff
@@ -1321,7 +1338,23 @@ pub struct ContextRef<'ctx> {
 }
 
 impl<'ctx> ContextRef<'ctx> {
-    pub(crate) unsafe fn new(context: LLVMContextRef) -> Self {
+    /// Get raw [`LLVMContextRef`].
+    ///
+    /// # Safety
+    ///
+    /// This function is exposed only for interoperability with other LLVM IR libraries.
+    /// It's not intended to be used by most users, hence marked as unsafe.
+    pub unsafe fn raw(&self) -> LLVMContextRef {
```

Comment:
Same here; calling the raw method alone can't produce undefined behavior since it just returns a pointer. Unsafety is in how it might be used